v2/AdminCore.Services/EmployeeService.cs

```diff
@@ -51,32 +51,61 @@ public bool VerifyEmailExists(string email)
 
     public IList<EmployeeDto> GetAll()
     {
-      throw new NotImplementedException();
-    }
-
-    public void Update(EmployeeDto employeeDto)
-    {
-      throw new NotImplementedException();
+      var employees = DatabaseContext.EmployeeRepository.Get();
+      return _mapper.Map<IList<EmployeeDto>>(employees);
     }
 
     public void Delete(int employeeId)
     {
-      throw new NotImplementedException();
+      var employeeDbEntry = GetEmployeeById(employeeId);
+      if (employeeDbEntry == null)
+      {
+        throw new Exception($"Employee could not be delete. Employee ID {employeeId} not found.");
+      }
+      DatabaseContext.EmployeeRepository.Delete(employeeDbEntry);
```

Comment:
Find out from @BrownMark if we ever want to delete an employee? Are should we soft delete them?